diff -ruN qt4.orig/src/plugins/AccessKeysNavigation/akn_handler.cpp qt4/src/plugins/AccessKeysNavigation/akn_handler.cpp
--- qt4.orig/src/plugins/AccessKeysNavigation/akn_handler.cpp	2014-01-29 05:31:09.000000000 +0400
+++ qt4/src/plugins/AccessKeysNavigation/akn_handler.cpp	2014-01-29 05:49:21.634522803 +0400
@@ -351,7 +351,7 @@
 void AKN_Handler::makeAccessKeyLabel(const QChar &accessKey, const QWebElement &element)
 {
     QLabel* label = new QLabel(m_view.data());
-    label->setText(QString(QLatin1String("<b>%1</b>")).arg(accessKey));
+    label->setText(QString(QLatin1String("<span style=\"color:#111111; font-weight:600;\">%1</span>")).arg(accessKey));
 
     QPalette p = QToolTip::palette();
     QColor color = QColor(220, 243, 253);
